example-app/SantokuApp/src/features/backend/utils/customInstance.ts

```diff
@@ -31,42 +32,94 @@ const getDefaultAxiosConfig = () => {
   } as AxiosRequestConfig;
 };
 
+/**
+ * 指定されたsignalを組み合わせて、一つのsignalを返却します。
+ *
+ * ＜参考＞
+ * MDNには、AbortSignal.anyというメソッドが存在していますが、React Nativeではまだ実装されていないようです。
+ * https://developer.mozilla.org/en-US/docs/Web/API/AbortSignal/timeout_static
```

Comment:
> MDNには、AbortSignal.anyというメソッドが存在していますが、React Nativeではまだ実装されていないようです。

リンク先だと↓のように書いてあって、`any`の検索結果が0件なんですが、書き換わったんですかね？
代わりに `browser_compatibility` を提案しておきます。

> Note: At time of writing there is no way to combine multiple signals. This means that you can't directly abort a download using either a timeout signal or by calling [AbortController.abort()](https://developer.mozilla.org/en-US/docs/Web/API/AbortController/abort).

[AbortSignal: timeout() static method - Web APIs | MDN](https://developer.mozilla.org/en-US/docs/Web/API/AbortSignal/timeout_static)


```suggestion
 * https://developer.mozilla.org/en-US/docs/Web/API/AbortSignal#browser_compatibility
```

Reply:
>リンク先だと↓のように書いてあって、anyの検索結果が0件なんですが、書き換わったんですかね？
代わりに browser_compatibility を提案しておきます。

いえ、URLの記載間違いですね。（提案してくれたURLと同じものを記載しようとしてました、、、）